eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOnext_line.java

```diff
@@ -59,7 +61,17 @@ public class EOstdin$EOnext_line extends PhDefault {
                                 "There is no line in the standard input stream to consume"
                             );
                         }
-                        return new Data.ToPhi(sc.nextLine());
+                        final String line = sc.nextLine();
+                        if (sc.hasNextLine()) {
+                            final StringBuilder leftlines = new StringBuilder();
```

Comment:
@includealex please, read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html